dropwizard-assets/src/main/java/io/dropwizard/assets/AssetsBundle.java

```diff
@@ -83,12 +85,31 @@ public AssetsBundle(String resourcePath, String uriPath, String indexFile) {
      * {@code resourcePath} of {@code "/assets"} and a uriPath of {@code "/js"},
      * {@code src/main/resources/assets/example.js} would be served up from {@code /js/example.js}.
      *
-     * @param resourcePath        the resource path (in the classpath) of the static asset files
-     * @param uriPath             the uri path for the static asset files
-     * @param indexFile           the name of the index file to use
-     * @param assetsName          the name of servlet mapping used for this assets bundle
+     * @param resourcePath the resource path (in the classpath) of the static asset files
+     * @param uriPath      the uri path for the static asset files
+     * @param indexFile    the name of the index file to use
+     * @param assetsName   the name of servlet mapping used for this assets bundle
      */
     public AssetsBundle(String resourcePath, String uriPath, String indexFile, String assetsName) {
+        this(resourcePath, uriPath, indexFile, assetsName, DEFAULT_MEDIA_TYPE);
+    }
+
+    /**
+     * Creates a new AssetsBundle which will configure the application to serve the static files
+     * located in {@code src/main/resources/${resourcePath}} as {@code /${uriPath}}. If no file name is
+     * in ${uriPath}, ${indexFile} is appended before serving. For example, given a
+     * {@code resourcePath} of {@code "/assets"} and a uriPath of {@code "/js"},
+     * {@code src/main/resources/assets/example.js} would be served up from {@code /js/example.js}.
+     *
+     * @param resourcePath     the resource path (in the classpath) of the static asset files
+     * @param uriPath          the uri path for the static asset files
+     * @param indexFile        the name of the index file to use
+     * @param assetsName       the name of servlet mapping used for this assets bundle
+     * @param defaultMediaType the default media type for unknown file extensions
+     * @since 2.0
+     */
+    public AssetsBundle(String resourcePath, String uriPath, String indexFile, String assetsName,
+                        String defaultMediaType) {
```

Comment:
This is getting to have quite a few parameters (and of the same type) so might not be that usable if we continue to add new parameters here. It might be nice to investigate using a builder to instantiate the bundle next time.